### PR TITLE
Add contents rust offical 

### PR DESCRIPTION
**Pull request by minatorak**
bisect rustc project is new -> https://github.com/rust-lang/cargo-bisect-rustc/
Const Generics -> https://github.com/rust-lang/project-const-generics

### CODE REVIEW

File: src/official.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Working groups:
 * [The Rust Language Design Team](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
 * [The Types Team](https://rust-lang.github.io/types-team/) - traits implementation improvements
 * [wg-async](https://rust-lang.github.io/wg-async/) - foundations of async I/O
+* [Const Generics Project Group](https://rust-lang.github.io/project-const-generics/index.html) - implements and designs the const_generics feature
```
Comment (glyn) [inline src/official.md:30]: Please could you move this to the top of that group so the book names are in alphabetical order?

File: src/official.md [hunk 2]
```diff
@@ -39,3 +40,4 @@ Other:
 * [Rust RFCs](https://rust-lang.github.io/rfcs/) - list of Requests For Comments for changes to Rust
 * [The Chalk Book](https://rust-lang.github.io/chalk/book/) - Rust's new trait system implementation
 * [The `bindgen` User Guide](https://rust-lang.github.io/rust-bindgen/) - automatically generates Rust FFI bindings to C and C++ libraries
+* [cargo bisect rustc](https://rust-lang.github.io/cargo-bisect-rustc/) - tool for automatically downloads rustc artifacts and tests until it finds the regression
```
Comment (glyn) [inline src/official.md:43]: Please could you move this so the book names in this group are in alphabetical order?

Also, please rename to `cargo-bisect-rust` to match the literal name of the book (compare `wg-async`).

**Review by glyn — CHANGES_REQUESTED** [review]
Thanks for this. Sorry for the slow response - I'm away from home at the moment.

Three simple bits of rework please.

**Comment by minatorak** [comment]
thank for review

File: src/official.md [hunk 3]
```diff
@@ -23,14 +23,15 @@ Advanced:
 
 Working groups:
 * [_Compiler team working groups_](https://rust-lang.github.io/compiler-team/working-groups/) - a list of working groups
+* [Const Generics Project Group](https://rust-lang.github.io/project-const-generics/index.html) - implements and designs the const_generics feature
```
Comment (glyn) [inline src/official.md:26]:
```suggestion
* [Const Generics Project Group](https://rust-lang.github.io/project-const-generics/index.html) - design and implementation of the const generics feature
```

File: src/official.md [hunk 4]
```diff
@@ -23,14 +23,15 @@ Advanced:
...
 * [Standard library developers Guide](https://std-dev-guide.rust-lang.org/about.html)
 * [The Rust Language Design Team](https://lang-team.rust-lang.org/design_notes.html) - information related to the work of the lang team
 * [The Types Team](https://rust-lang.github.io/types-team/) - traits implementation improvements
 * [wg-async](https://rust-lang.github.io/wg-async/) - foundations of async I/O
-* [Const Generics Project Group](https://rust-lang.github.io/project-const-generics/index.html) - implements and designs the const_generics feature
 
 Other:
 * [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book/) - non-blocking coroutines
+* [cargo-bisect-rustc](https://rust-lang.github.io/cargo-bisect-rustc/) - tool for automatically downloads rustc artifacts and tests until it finds the regression
```
Comment (glyn) [inline src/official.md:34]:
```suggestion
* [cargo-bisect-rustc](https://rust-lang.github.io/cargo-bisect-rustc/) - tool for automatically downloading and bisecting rustc artifacts and tests until it finds a given regression
```

**Review by glyn — APPROVED** [review]
Looks good, thanks. Just tweaked the text a little.

**Comment by glyn** [comment]
Thanks again @minatorak!

**Pull request merged**